java/runtime/src/main/java/io/ray/runtime/object/ObjectStore.java

```diff
@@ -99,9 +99,11 @@ public void put(Object object, ObjectId objectId) {
         object = ObjectSerializer
             .deserialize(dataAndMeta, ids.get(i), elementType);
       }
-      if (object instanceof RayException) {
+      if (object instanceof io.ray.api.exception.RayException) {
```

Comment:
Why do we have both `io.ray.api.exception.RayException` and `io.ray.runtime.exception.RayException`?

Reply:
The old `RayException` logic is not changed in this PR.